docker-compose.yml

```diff
@@ -1,18 +1,18 @@
-version: "3.4"
+version: "3.5"
 services:
   bind.dnp.dappnode.eth:
     build: ./build
     image: "bind.dnp.dappnode.eth:0.2.6"
     container_name: DAppNodeCore-bind.dnp.dappnode.eth
     restart: always
     networks:
-      network:
+      dncore_network:
         ipv4_address: 172.33.1.2
+        aliases:
+          - bind.dappnode
```

Comment:
Maybe alias name should be something else since we will be removing bind from this package.